chargeio/Program.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Microsoft.Extensions.DependencyInjection;  
+using Microsoft.Extensions.Logging;
+using chargeio;
+
+public class Program
```

Comment:
What is Program.cs for, in a library package like this? Is this supposed to be here? I understand in a sample app, etc. But in the main library code?